Remotion/Web/ClientScript/Scripts/SmartPage.ts

```diff
@@ -1281,19 +1281,3 @@ class SmartPage_Context
   // The single instance of the SmartPage_Context object
   public static Instance: Nullable<SmartPage_Context> = null; // TODO RM-7696: Convert SmartPage_Context.Instance to a get method and throw an exception when it is called before initialization
 }
-
-// Called after page's html content is complete.
-function SmartPage_OnStartUp(isAsynchronous: boolean, isDirty: boolean): void
-{
-  SmartPage_Context.Instance!.OnStartUp(isAsynchronous, isDirty);
-}
-
-function SmartPage_PageRequestManager_pageLoaded(sender: Sys.WebForms.PageRequestManager, args: Sys.WebForms.PageLoadedEventArgs): void
-{
-  SmartPage_Context.Instance!.PageRequestManager_pageLoaded(sender, args);
-}
-
-function RenderThisHtml(theHtml: string): void
```

Comment:
this needs a separate breaking change doc. ages ago, we added this as a utility function to deal with ActiveX.

Reply:
You mean a separate breaking change ticket for the removal of `RenderThisHtml`? Where was this function even used? I could not find any usages in JS or C#

Reply:
Exactly. This function wasn't used by re-motion but downstream. It may well be that the last usages have been removed by now anyway, but we need to do a breaking change doc. And given that it stood appart from the rest of the APIs, a separate ticket is helpful.